src/app/client/src/app/modules/recover-account/components/verify-account-identifier/verify-account-identifier.component.html

```diff
@@ -25,11 +25,11 @@
     <re-captcha *ngIf="isP2CaptchaEnabled === 'true'" #captchaRef="reCaptcha" (resolved)="$event && resolved($event)" siteKey="{{googleCaptchaSiteKey}}" size="invisible"></re-captcha>
         <a (click)="captchaRef.reset()" id="resetGoogleCaptcha"></a>
     <div class="d-flex flex-jc-space-between mt-32">
-      <button class="sb-btn sb-btn-normal width-100 mr-4 sb-btn-outline-primary" tabindex="0" [ngClass]="{'sb-btn-disabled': !disableResendOtp}"
+      <button class="sb-btn sb-btn-normal width-100 mr-4 sb-btn-outline-primary content-align" tabindex="0" [ngClass]="{'sb-btn-disabled': !disableResendOtp}" [ngStyle]="{'font-size': resendOTPbtn && resendOTPbtn.includes('OTO ஐ  மீண்டும் அனுப்பு') ? '.6rem' : '.75rem' }"
```

Comment:
cannot add ngStyle like this please speak to the css team and work on this issue

Reply:
Ok